tests/test_coils.py

```diff
@@ -533,6 +489,66 @@ def test_coilset_convert(self):
         np.testing.assert_allclose(B3, B4, rtol=1e-2)
 
 
+@pytest.mark.unit
+def test_symmetry_position(DummyCoilSet):
+    """Tests that compute position is correct from symmetry."""
+    # same coil sets with vs without symmetry
+    coilset_sym = load(
+        load_from=str(DummyCoilSet["output_path_sym"]), file_format="hdf5"
+    )
+    coilset_asym = load(
+        load_from=str(DummyCoilSet["output_path_asym"]), file_format="hdf5"
+    )
+    coilset_mixed = MixedCoilSet(*coilset_asym)
+    grid = LinearGrid(N=30)
+
+    # check that positions of CoilSets are the same with xyz basis
+    x_sym = coilset_sym._compute_position(basis="xyz", grid=grid)
+    x_asym = coilset_asym._compute_position(basis="xyz", grid=grid)
+    x_mixed = coilset_mixed._compute_position(basis="xyz", grid=grid)
+
+    np.testing.assert_allclose(x_sym, x_asym)
+    np.testing.assert_allclose(x_sym, x_mixed)
+
+    # check that positions of CoilSets are the same with rpz basis
+    x_sym = coilset_sym._compute_position(basis="rpz", grid=grid)
+    x_asym = coilset_asym._compute_position(basis="rpz", grid=grid)
+    x_mixed = coilset_mixed._compute_position(basis="rpz", grid=grid)
+
+    np.testing.assert_allclose(x_sym, x_asym)
+    np.testing.assert_allclose(x_sym, x_mixed)
+
+
+@pytest.mark.unit
+def test_symmetry_magnetic_field(DummyCoilSet):
+    """Tests that compute magnetic field is correct from symmetry."""
+    # same coil sets with vs without symmetry
+    eq = get("precise_QH")
+    coilset_sym = load(
+        load_from=str(DummyCoilSet["output_path_sym"]), file_format="hdf5"
+    )
+    coilset_asym = load(
+        load_from=str(DummyCoilSet["output_path_asym"]), file_format="hdf5"
+    )
+
+    # test that both coil sets compute the same field on the plasma surface
```

Comment:
I'm confused what the actual difference between the coilsets is. If one doesn't have symmetry then why do we expect it produce the same field?

Reply:
They are physically the same sets of coils. But the "asymmetric" one includes all of the coils explicitly, and the "symmetric" one only has coils in half of a field period and uses virtual coils from the symmetry assumption.

Reply:
Hmm okay, so it's not that it's actually asymmetric when created right? It's more so that the optimizer doesn't know about any symmetries within the "asymmetric coilset"?

Reply:
yea exactly